lib/ipmi_backend_utils.pm

```diff
@@ -215,14 +266,22 @@ sub set_serial_console_on_xen {
 
```

Comment:
The function name is set_serial_console_on_xen, but you plan to support kvm in it?

Reply:
Yes. IF..ELSE logic in it does this.

Reply:
I mean you can not put kvm serial console in this function set_serial_console_on_xen which means only xen console setting should be in.